.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  release:
+    types: [prereleased, released]
+
+jobs:
+  release-docker:
+    name: Release docker & config
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: roobre/k8s-metadata-injection # change this!
+      DOCKER_IMAGE_TAG: ${{ github.event.release.tag_name }}
+      S3_PATH: s3://nr-downloads-main/infrastructure_agent
+    steps:
+      - name: Validate tag name
+        run: echo "$DOCKER_IMAGE_TAG" | grep -E '^v[0-9.]*[0-9]$' > /dev/null
```

Comment:
I thought this could be a good idea, to avoid pushing weird stuff by accident

Reply:
we ususally don't, but then, it also doesn't hurt 👍